### PR TITLE
feat: add open and close delay props to tooltip

### PR DESCRIPTION
**Pull request by jesperwiner**

### CODE REVIEW

File: src/Molecules/Tooltip/hooks/useTooltip.ts [hunk 1]
```diff
@@ -24,19 +24,20 @@ export const useTooltip = (mode: TooltipProps['mode']) => {
 
   const handleMouseEnter = useCallback(() => {
     if (mode !== 'hover') return;
+    console.log('store', store);
```
Comment (Vikingjonsson) [inline src/Molecules/Tooltip/hooks/useTooltip.ts:27]: log

**Review by kmelkon — APPROVED** [review]

File: src/Molecules/Tooltip/hooks/useTooltip.ts [hunk 2]
```diff
@@ -24,19 +24,20 @@ export const useTooltip = (mode: TooltipProps['mode']) => {
...
     if (!store.isVisible()) {
-      store.startBecomingVisibleTimeout(id);
+      store.startBecomingVisibleTimeout(id, openDelay);
     } else if (store.state === State.LEAVING_VISIBLE) {
       store.setState(State.VISIBLE, id);
     }
-  }, [id, mode]);
+  }, [id, mode, openDelay]);
 
   const handleMouseMove = useCallback(() => {
```
Comment (Vikingjonsson) [inline src/Molecules/Tooltip/hooks/useTooltip.ts:36]: if this should act the same as handleMouseEnter (maybe enough with one of them) then is probably good to have the same dependencies.

File: src/Molecules/Tooltip/hooks/useTooltip.ts (hunk 1 above)
Comment (jesperwiner) [reply src/Molecules/Tooltip/hooks/useTooltip.ts]: removed

File: src/Molecules/Tooltip/hooks/useTooltip.ts (hunk 2 above)
Comment (jesperwiner) [reply src/Molecules/Tooltip/hooks/useTooltip.ts:35]: fixed

**Review by Vikingjonsson — APPROVED** [review]

**Pull request merged**